src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Utf8JsonReader.SingleSegment.Tests.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using MicroBenchmarks.Serializers;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public partial class Utf8JsonReaderTests
+    {
+        private const int Iterations = 1_000_000;
+
+        private byte[] s_singleEmptyMultiLineComment;
+        private byte[] s_multipleEmptyMultiLineComment;
+        private byte[] s_singleShortMultiLineComment;
+        private byte[] s_multipleShortMultiLineComment;
+        private byte[] s_singleLongMultiLineComment1;
+        private byte[] s_singleLongMultiLineComment2;
+
+        private byte[] s_singleEmptySingleLineComment;
+        private byte[] s_multipleEmptySingleLineComment;
+        private byte[] s_singleShortSingleLineComment;
+        private byte[] s_multipleShortSingleLineComment;
+        private byte[] s_singleLongSingleLineComment;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            // every test case ends with {} so that we can re-use it for both Skip and Allow options
+            s_singleEmptyMultiLineComment = Encoding.UTF8.GetBytes("/**/{}");
+            s_multipleEmptyMultiLineComment = Encoding.UTF8.GetBytes("/**//**//**//**//**//**//**//**//**//**//**//**//**//**//**//**//**//**/{}");
+            s_singleShortMultiLineComment = Encoding.UTF8.GetBytes("/*asdasd*/{}");
+            s_multipleShortMultiLineComment = Encoding.UTF8.GetBytes("/*asdasd*//*asdasd*//*asdasd*//*asdasd*//*asdasd*//*asdasd*//*asdasd*//*asdasd*//*asdasd*//*asdasd*/{}");
+            s_singleLongMultiLineComment1 = Encoding.UTF8.GetBytes("/*" + new string('c', 2000) + "*/{}");
+            s_singleLongMultiLineComment2 = Encoding.UTF8.GetBytes("/*" + new string('\n', 2000) + "*/{}");
+
+            s_singleEmptySingleLineComment = Encoding.UTF8.GetBytes("//\n{}");
+            s_multipleEmptySingleLineComment = Encoding.UTF8.GetBytes("//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n//\n{}");
```

Comment:
I think an array of JSON elements with comments in between is a better test case than lots of comments together (particularly for skip), since that ends up creating a tight loop while reading/skipping comments which is unlikely to happen for any realistic payload.

So, something like:
`[1//\n,2//\n,3//\n,...]`